examples/native-fungible/src/contract.rs

```diff
@@ -144,6 +144,9 @@ impl NativeFungibleTokenContract {
                 );
             }
             AccountOwner::Application(_) => panic!("Applications not supported yet"),
+            AccountOwner::Chain => {
+                panic!("Chains cannot be used to authenticate")
```

Comment:
`"Chain accounts are not supported"`

Reply:
Done in [Replace unreachables with panics](https://github.com/linera-io/linera-protocol/pull/3560/commits/f8d5e33ec6a0610a99a2b4a9524e399cf37e60c4).